tests/test_incorrect_base_class.py

```diff
@@ -0,0 +1,54 @@
+"""Tests for incorrect base class checker."""
+from pathlib import Path
+
+import astroid
+from pylint.testutils import CheckerTestCase
+from pylint.testutils import MessageTest
+from pytest import mark
+
+from pylint_nautobot.incorrect_base_class import NautobotIncorrectBaseClassChecker
+
+from .utils import assert_no_message
+
+_INPUTS_PATH = Path(__file__).parent / "inputs/incorrect-base-class"
```

Comment:
Perhaps? I'm not real familiar with the Path API yet.

```suggestion
_INPUTS_PATH = Path(__file__).parent / "inputs" / "incorrect-base-class"
```